Source/WebCore/editing/Editor.cpp

```diff
@@ -591,7 +591,7 @@ bool Editor::deleteWithDirection(SelectionDirection direction, TextGranularity g
 
     if (m_document.selection().isRange()) {
         if (isTypingAction) {
-            TypingCommand::deleteKeyPressed(document(), canSmartCopyOrDelete() ? TypingCommand::SmartDelete : 0, granularity);
+            TypingCommand::deleteKeyPressed(document(), canSmartCopyOrDelete() ? TypingCommand::Option::SmartDelete : OptionSet<TypingCommand::Option> { }, granularity);
```

Comment:
maybe try `{ }` instead of `OptionSet<TypingCommand::Option> { }`? (or, if that doesn't work, `OptionSet { }` maybe)

Reply:
It seems that neither work, so I think I have to keep what was originally there.